python/ray/tests/test_cython.py

```diff
@@ -54,4 +54,6 @@ def test_numpy(self):
 
 
 if __name__ == "__main__":
-    unittest.main(verbosity=2)
+    import pytest
```

Comment:
I switched to using pytest -v for all tests since it only prints erroring tests, which goes well with --test_output=errors in bazel.